### PR TITLE
Neural Network - Base

### PR DESCRIPTION
**Pull request by Abheeshth**
I have added Neural Network  base model file please let me know if there are any changes I should do
@geekquad 
issue #47

### CODE REVIEW

**Review by geekquad — CHANGES_REQUESTED** [review]
Hey @Abheeshth, did you went through the Contributing Guidelines?
We are not accepting codes in Jupyter Notebook.
Please make the necessary changes!

**Comment by Abheeshth** [comment]
sure sir I will  do some necessary changes with in 2 days

File: Neural Network/neural_network.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+"""Neural Network.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1HMxwkMbHBiP3DnVkR59bFIwQ7WLo8S51
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
```
Comment (geekquad) [inline Neural Network/neural_network.py:11]: matplotlib is not being used anywhere. Please omit this line and we are good to go!

**Review by geekquad — CHANGES_REQUESTED** [review]

File: Neural Network/neural_network.py (hunk 1 above)
Comment (Abheeshth) [reply Neural Network/neural_network.py]: Done it

**Review by geekquad — APPROVED** [review]
Good work @Abheeshth.

**Pull request merged**